### PR TITLE
Reword sending paused message for Creator plans [MAILPOET-6026]

### PR DESCRIPTION
**Pull request by alex-mpoet**
## Description

_N/A_

## Code review notes

_N/A_

## QA notes

Testing instructions:
1. Verify a Creator API key on your test site. If you had another plan before it, you may need to clear the `mta.mailpoet_api_key_state` setting manually if it retains the `valid` state.
2. Select MSS as sending method.
3. Observe the message, check links.

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-6026]

## After-merge notes

_N/A_

## Tasks

- [ ] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [ ] I added sufficient test coverage
- [ ] I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

[MAILPOET-6026]: https://mailpoet.atlassian.net/browse/MAILPOET-6026?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/assets/js/src/notices/invalid-mss-key-notice.tsx [hunk 1]
```diff
@@ -4,24 +4,56 @@ import ReactStringReplace from 'react-string-replace';
...
+        {premiumKeyValid &&
+          ReactStringReplace(
+            MailPoet.I18n.t('allSendingPausedPremiumValidBody'),
+            /\[link\](.*?)\[\/link\]/g,
+            (match, index) => {
+              // first link tag
+              if (index === 1) {
+                return (
+                  <a
+                    href={`https://account.mailpoet.com?s=${subscribersCount}`}
```
Comment (websupporter) [inline mailpoet/assets/js/src/notices/invalid-mss-key-notice.tsx]: Would it make sense to use an upgrade link instead? `Kindly upgrade to a [link]MailPoet sending plan[/link]`?

**Comment by websupporter** [comment]
I am currently looking into [MAILPOET-6028] and wonder, if this modal should be shown globally. What do you think @alex-mpoet? 

[MAILPOET-6028]: https://mailpoet.atlassian.net/browse/MAILPOET-6028?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

File: mailpoet/assets/js/src/notices/invalid-mss-key-notice.tsx (hunk 1 above)
Comment (alex-mpoet) [reply mailpoet/assets/js/src/notices/invalid-mss-key-notice.tsx]: Yes, this makes sense. What UTM params should this link have? Same as on the Premium upgrade page?

**Comment by alex-mpoet** [comment]
@websupporter 
>wonder, if this modal should be shown globally

It's currently shown on several pages (Homepage, Newsletters, Segments, Subscribers), but not everywhere, most notably not in Settings. I can show it in Settings or show it everywhere, whatever works for you.

File: mailpoet/assets/js/src/notices/invalid-mss-key-notice.tsx (hunk 1 above)
Comment (websupporter) [reply mailpoet/assets/js/src/notices/invalid-mss-key-notice.tsx]: Mostly. The ref should change though to something that specifies the origin like `invalid_mss_key_notice` or something else.

**Comment by websupporter** [comment]
> It's currently shown on several pages (Homepage, Newsletters, Segments, Subscribers), but not everywhere, most notably not in Settings. I can show it in Settings or show it everywhere, whatever works for you.

Thanks for asking. With "globally" I meant the main MailPoet pages, not the whole wp-admin.

**Comment by alex-mpoet** [comment]
@websupporter Changes made.

**Comment by websupporter** [comment]
Thank you @alex-mpoet. Sorry for the delay. LGTM

**Review by Aschepikov — APPROVED** [review]
GJ!

**Pull request merged**